src/db/spanner/models.rs

```diff
@@ -996,7 +1139,14 @@ impl SpannerDb {
         .params(sqlparams)
         .execute_dml_async(&self.conn)
         .await?;
-        self.touch_collection_async(&params.user_id, collection_id)
+        let mut tags = Tags::default();
+        tags.tags
+            .insert("collection_id".to_string(), collection_id.to_string());
+        self.metrics
+            .incr_with_tags("self.storage.delete_bsos", Some(tags));
+        self.calc_quota_usage_async(&params.user_id, collection_id)
```

Comment:
Not necessarily a request but I think preferably the tombstone collection wouldn't be involved in calculating a quota for itself, I think it's a bit confusing.

Reply:
I mostly didn't want multiple writes, if possible.